emission/net/ext_service/otp/otp.py

```diff
@@ -30,8 +28,9 @@ def __init__(self, start_point, end_point, mode, date, time, bike, max_walk_dist
         self.accepted_modes = {"CAR", "WALK", "BICYCLE", "TRANSIT"}
         self.start_point = start_point
         self.end_point = end_point
-        if mode not in self.accepted_modes:
-            raise Exception("You are using a mode that doesnt exist")
+        # if mode not in self.accepted_modes:
```

Comment:
I'm spelling bicycle wrong somewhere and its causing problems ...
This isnt the first time ive made this mistake either